internal/gke/metrics.go

```diff
@@ -114,6 +116,25 @@ func (m *metricExporter) Close() error {
 	return m.client.Close()
 }
 
+// filterMetrics returns a set of filtered metrics.
+func filterMetrics(snaps []*metrics.MetricSnapshot, exportGeneratedMetrics bool) []*metrics.MetricSnapshot {
```

Comment:
We need to let the metric used during rollouts go through. This is annoying, we can chat about it in office.

Reply:
Yeah, I already did that change yesterday: https://github.com/ServiceWeaver/weaver/blob/main/http.go#L96. For the httpRequestCount, we don't label it as generated.

Reply:
ACK!